community/contributing/ways_to_contribute.rst

```diff
@@ -84,7 +84,7 @@ To ensure good collaboration and overall quality, the Godot developers
 enforce some rules for code contributions, for example regarding the style to
 use in the C++ code (indentation, brackets, etc.) or the git and PR workflow.
 
-A nice place to start may be the issue tagged as `junior jobs <https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+label%3A%22junior+job%22>`_ on GitHub.
+A good place to start is by searching for issues tagged as `junior jobs <https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+label%3A%22junior+job%22>`_ (or Hacktoberfest during October) on GitHub.
```

Comment:
You should link `Hacktoberfest` to the `Hacktoberfest` label since it's just a renamed `junior job` label.

Reply:
Done